tests/codemods/test_numpy_nan_equality.py

```diff
@@ -0,0 +1,75 @@
+from core_codemods.numpy_nan_equality import NumpyNanEquality
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from textwrap import dedent
+
+
+class TestNumpyNanEquality(BaseCodemodTest):
+    codemod = NumpyNanEquality
+
+    def test_name(self):
+        assert self.codemod.name() == "numpy-nan-equality"
+
+    def test_simple(self, tmpdir):
+        input_code = """\
+        import numpy
+        if a == numpy.nan:
+            pass
+        """
+        expected = """\
+        import numpy
+        if numpy.isnan(a):
+            pass
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_alias(self, tmpdir):
+        input_code = """\
+        import numpy as np
+        if a == np.nan:
+            pass
+        """
+        expected = """\
+        import numpy as np
+        if np.isnan(a):
+            pass
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_multiple_comparisons(self, tmpdir):
```

Comment:
I know it's sort of implicit in this and the next test case but it would be good to have an explicit test case with `np.nan` on the LHS of the comparison.